server/scripts/populateDB.js

```diff
@@ -0,0 +1,115 @@
+// to run script:
+// 1. cd server
+// 2. yarn populate:db
+const mongoose = require('mongoose')
+const tripSchema = require('../models/TripModel')
+const stageSchema = require('../models/StageModel')
+const userSchema = require('../models/UserModel')
+const { ObjectId } = require('mongodb')
+const config = require('../config/config')
+
+// Create models
+const Trip = mongoose.model('Trip', tripSchema.schema)
+const Stage = mongoose.model('Stage', stageSchema.schema)
+const User = mongoose.model('User', userSchema.schema)
+
+const populateDB = async () => {
+  try {
+    const conString = config.mongo.uri
+
+    // Connect to Mongo DB
+    await mongoose.connect(conString, {
+      useNewUrlParser: true,
+      useUnifiedTopology: true,
+    })
+    console.log('Connected to database...')
+
+    // Delete all records from trips, stages and users collections
+    await Trip.deleteMany({})
+    await Stage.deleteMany({})
+    await User.deleteMany({})
+
+    // Insert a user
+    const savedUser = await new User({
+      username: 'test',
+      email: 'passwordis@pass.com',
+      password: 'pass',
```

Comment:
this will not pass the verification backend btw, because the pass is expected to be hashed with bcrypt

Reply:
otherwise everything looks good